projects/valtimo/dossier/src/lib/components/dossier-detail/dossier-detail.component.html

```diff
@@ -236,7 +257,7 @@
       <div class="buttons-container">
         <cds-overflow-menu
           [ngClass]="{'--compact': compactMode$ | async}"
-          *ngIf="obs.canHaveAssignee"
+          *ngIf="obs.canHaveAssignee || obs.canDelete"
```

Comment:
What happens if it can't have an Assignee but you can delete the case? The unassign button would still be shown